Telegram bot

```diff
@@ -0,0 +1,256 @@
+
+        
+    else:
+        bot.send_message(cid, 'Try /new, /modify, /list or /notifications')
+
+def timecheck(time):
+    try:
+        datetime.strptime(time, '%H:%M') #I check if the user has read the instruction
+        return True
+    except Exception:
+        return False
+
+def datecheck(date):
+    try:
+        datetime.strptime(date, '%d.%m.%Y')
+        return True
+    except Exception:
+        return False    
+    
+def get_name(message): 
+    global name;
+    name = message.text;
+    bot.send_message(message.from_user.id, 'Type the date of event as DD.MM.YYYY'); #instruction itself
+    bot.register_next_step_handler(message, get_date);
+
+def get_date(message):
+    global date;
+    if datecheck(message.text)==True:
+        date = message.text;
+        bot.send_message(message.from_user.id, 'Type the time of event as HH:MM, 24-hours format');
+        bot.register_next_step_handler(message, get_time);
+    else:
+        bot.send_message(message.from_user.id, 'I can not read it...try again in DD.MM.YYYY');
+        bot.register_next_step_handler(message, get_date);        
+
+def get_time(message):
+    global time
+    if timecheck(message.text)==True:
+        time = message.text;
+        keyboard = types.InlineKeyboardMarkup(); 
+        key_yes = types.InlineKeyboardButton(text='Yes', callback_data='yes'); 
+        keyboard.add(key_yes); 
+        key_no= types.InlineKeyboardButton(text='No', callback_data='no');
+        keyboard.add(key_no);
+        question = 'The event is '+name+' on '+date+' at '+time+'?'
+        bot.send_message(message.from_user.id, text=question, reply_markup=keyboard)
+    else:
+        bot.send_message(message.from_user.id, 'I can not read it...try again in HH:MM');
+        bot.register_next_step_handler(message, get_time); 
+
+
+    
+def change_name(message): 
+    temp=Plans.create(uid=cid,event=ename,date=edate,time=etime,notification=Plans.get(uid=cid).notification, flag=eflag) #e-variables are from the entity we "modify"
+    Plans.get(uid=cid,event=ename,date=edate,time=etime).delete_instance()
+    temp.event=message.text
+    temp.save()
+    markup = telebot.types.ReplyKeyboardRemove()
+    bot.send_message(cid,'The name has been changed!', reply_markup=markup);    
+
+def change_date(message):
+    if datecheck(message.text)==True:
+        temp=Plans.create(uid=cid,event=ename,date=edate,time=etime,notification=Plans.get(uid=cid).notification, flag=0)
+        Plans.get(uid=cid,event=ename,date=edate,time=etime).delete_instance()
+        temp.date=message.text
+        temp.save()
+        markup = telebot.types.ReplyKeyboardRemove()
+        bot.send_message(cid,'The date has been changed!', reply_markup=markup); 
+    else:
+        bot.send_message(message.from_user.id, 'I can not read it...try again in DD.MM.YYYY');
+        bot.register_next_step_handler(message, get_date);   
+    
+
+
+def change_time(message):
+    if timecheck(message.text)==True:
```

Comment:
Code duplication with `change_date`
You don't have to write `== True`, I suppose the expression can be simplified